open-metadata-resources/open-metadata-deployment/docker/egeria/Dockerfile

```diff
@@ -32,6 +32,7 @@ ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/server-chassis-spring/${versio
 ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/ui-chassis-spring/${version}/maven-metadata.xml /opt/egeria/user-chassis-spring.xml
 ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/server-chassis-spring/${version}/server-chassis-spring-${version}.jar /opt/egeria/server-chassis-spring.jar
 ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/ui-chassis-spring/${version}/ui-chassis-spring-${version}.jar /opt/egeria/ui-chassis-spring.jar
+ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/open-lineage-janus-connector/${version}/open-lineage-janus-connector-${version}-jar-with-dependencies.jar /opt/egeria/lib/janus-connector-jar-with-dependencies.jar
```

Comment:
Not critical, but to follow the same pattern perhaps you could add the .xml file (has version identifier) - similar to what is done for the ui-chassis & chassis?

Reply:
perfect - thanks for adding